### ISSUE TITLE
File name on the bottom of the image?

### ISSUE THREAD

**Issue by wochen** [dawnlabs/alchemy#54]
Hi there!

Is there any chance you can add (as an option) the file name - to be burned in somewhere on the bottom of the image (during converting jpg(s) to pdf)?

Thanks.

**Comment by mfix22** [comment]
I don't think we are trying to support that here, as there are already Mac/Windows native apps that can handle this feature. We are primarily focused on converting/merging files
